cron.d/sbws

```diff
@@ -0,0 +1,19 @@
+# Example crontab for sbws
+
+# Edit SBWSBIN if sbws executable is in other location
+SBWSBIN=/usr/local/bin/sbws
+
+# Edit SBWSHOME if sbws home directory is in other location
+SBWSHOME=$HOME/.sbws
+
+# Bandwidth list files should be generated and transfer between 
+# 5-15 or 35-45 minutes past the hour
+# The following commented lines are all the possible times at which sbws 
+# should run. Choose only one.
+# 5 * * * *
+# 10 * * * *
+# 15 * * * *
+# 35 * * * *
+# 40 * * * *
+# 45 * * * *
+40 * * * *    $SBWSBIN -d $SBWSHOME generate
```

Comment:
Do crontab files need to end in a newline?

Reply:
I'll merge as soon as @juga0 answers and, if necessary, fixes.

Reply:
not AFAIK, but i did not find documentation about it that confirms it.
I tested it works.